packages/yew/tests/pr_2705.rs

```diff
@@ -0,0 +1,80 @@
+#![cfg(target_arch = "wasm32")]
+
+mod common;
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+use std::time::Duration;
+
+use common::obtain_result;
+use gloo::timers::future::sleep;
+use wasm_bindgen_futures::spawn_local;
+use wasm_bindgen_test::*;
```

Comment:
try to avoid wildcard imports if you can

Reply:
Negligeable gains from changing this in test code, it's in the other test files as well, anyway.